app/src/main/res/values/strings.xml

```diff
@@ -780,4 +804,10 @@
     <string name="main_drawer_close">Close</string>
     <string name="main_drawer_help">Help</string>
     <string name="main_drawer_login">Log in / join Wikipedia</string>
+    <string name="custom_date_picker_dialog_ok_button_text">OK</string>
+    <string name="text_input_dialog_ok_button_text">OK</string>
+    <string name="ok">OK</string>
```

Comment:
Not sure why don't move the strings that relate to developer preferences to `dev_settings_strings.xml`?

Reply:
Done